coffee-shop/package.json

```diff
@@ -16,7 +16,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
```

Comment:
you will noy need to use json server, as you will use the mock data direct in utils functions, I think we don't need this PR

Reply:
ok